addons/website_sale_stock/views/website_sale_stock_templates.xml

```diff
@@ -50,4 +50,12 @@
         </t>
     </xpath>
   </template>
+
+  <template id="products_add_to_cart_out_of_stock" inherit_id="website_sale.products_add_to_cart">
+    <xpath expr="//div[hasclass('o_wsale_product_btn')]" position="replace">
+      <t t-set="info" t-value="product.with_context(website_sale_stock_get_quantity=True)._get_combination_info()"></t>
+      <t t-if="info['product_type'] == 'product' and not info['allow_out_of_stock_order'] and not (info['free_qty'] - info['cart_qty'])"></t>
+      <t t-else="1">$0</t>
```

Comment:
<t t-else="">$0</t>

Reply:
From de documentation:

> `replace`
replaces the matched element with the xpath’s body, replacing any `$0` node occurrence in the new body with the original element
> [View Inheritance](https://www.odoo.com/documentation/15.0/developer/howtos/rdtraining/13_inheritance.html?highlight=position#view-inheritance)